examples/push-repo

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import kobo.rpmlib
+import os
+
+from chexus import Client, BucketItem, TableItem
+
+LOG = logging.getLogger("push-repo")
+
+
+def get_sigkey(rpmfile):
```

Comment:
You should be able to share these rather than duplicating by example via something like:

- in examples directory, create ``__init__.py`` file (it can be empty)
- in examples directory, create ``utils.py`` file and put these functions in them
- in each of the examples, do ``from .utils import get_sigkey, cdn_path``

May require some playing around to make it work.  I wouldn't say it's necessary since they are only examples.

Reply:
> May require some playing around to make it work. 

Indeed it does. I'm still working on this one. :)

Reply:
I could only import utils.py as a package if I created a subdir for utils.py. So, instead, I did not package utils.py and imported it unpackaged. Because this is just an example script, I don't see a problem with this, but let me know if I'm missing something.